BUILD.gn

```diff
@@ -137,6 +137,26 @@ npm_action("atom_browserify_isolated") {
   ]
 }
 
+npm_action("atom_browserify_content_script") {
+  script = "browserify"
+
+  inputs = [
+    "lib/content_script/init.js",
+  ]
+
+  outputs = [
+    "$target_gen_dir/js2c/content_script_bundle.js",
+  ]
+
+  args = [
+    "lib/content_script/init.js",
+    "-t",
+    "aliasify",
```

Comment:
Although it doesn't need it right now, it will in the future, can we include the `tsify` setup from the other browerify configs in this `args` and `deps` list 👍

Reply:
8d020154d4d7ebb2deb13b44ff0555baa2b39a13